src/client/viewSandbox.js

```diff
@@ -117,6 +117,13 @@ const reportIframeCommsError = error => {
 };
 
 const init = () => {
+  // Hide link to the library sandbox for non web extensions.
+  if (
+    !extensionDescriptor || !extensionDescriptor.platform || extensionDescriptor.platform !== 'web'
```

Comment:
It looks like lower in the file we have one place that does:

```
const categorizedItems = getCategorizedItems(extensionDescriptor[groupKey]);
```

and another that does:

```
if (extensionDescriptor.configuration) {
```

which don't check for the existence of `extensionDescriptor`, which leads me to believe that `extensionDescriptor` is probably always available and we don't need to do a check for it. We can/should probably also remove this other check that's also in this file:

```
if (extensionDescriptor) {
```

Reply:
Done.